### DISCUSSION TITLE
Breakout room 1

### DISCUSSION THREAD

**Discussion by joemccallum** [IBAHCM/RPiR#265] [Archive]
Joe McCallum - @joemccallum
Loyce Majige - @LoyceMajige
Ismail Nambunga - @Nambunga

**Comment by joemccallum** [comment]
Joe McCallum Repo - https://github.com/IBAHCM/joemccallumBCI.git

**Reply by Nambunga** [reply]
@joemccallum Hello! I went through your repository and run the demo. It's working well for me and report was compiled perfectly.

**Comment by LoyceMajige** [comment]
Loyce Majige repo - https://github.com/IBAHCM/LoyceMajigeBCI.git

**Comment by Nambunga** [comment]
Ismail Nambunga repo - https://github.com/IBAHCM/NambungaBCI

**Comment by Nambunga** [comment]
@joemccallum @LoyceMajige Could you please have a look at my repository and see if the demo is working well?
best regards

**Comment by LoyceMajige** [comment]
Hi @joemccallum, @Nambunga 

I have push my work to GitHub,
I would really appreciate if you guys can check the demos and reports on your device when you have time!
Please let me know if you notice something or find error messages.

Thank you!
Loyce :)

**Reply by Nambunga** [reply]
@LoyceMajige Hi! I was looking at your package and worked fine for me. Even report was successful compiled.

**Reply by LoyceMajige** [reply]
Cool beans! Thanks
